Kyle-Law/rails-tracker-api
# Merging Feature-branch into develop

# Portfolio Tracker API

> Rails Backend API for [rails-react-tracker](https://github.com/Kyle-Law/react-rails-tracker)

## Models

This API is made of two models

- user model
- measurement model

## Endpoints

GET /users to return all users

POST /users to add a new user

GET /measurements to returns all the measurements

POST /measurements to add a new measurement

## ERD Diagram

![image](https://user-images.githubusercontent.com/55923773/93226288-bbc2ed00-f7a5-11ea-9d0b-83064fc83207.png)

## Testing

1. Clone the project and cd to private-events
2. Run `yarn install --check-files` to check and update yarn packages
3. Run 'rspec'
4. 3 examples, 0 failures should be shown.

:)
